postscripts/puppet_config_cached_catalog

```diff
@@ -0,0 +1,104 @@
+#!/bin/bash
+
+###
+# Description: Configure puppet agent use_cached_catalog setting.
+#              Expects that puppet agent is already installed and configured.
+#              See also: postscripts/puppet_configure (in this repo)
+# Usage:
+#      1. updatenode <noderange> -P "puppet_config_cached_catalog [OPTS]"
+#
+# Source: https://github.com/ncsa/xcat-tools
+###
+
+
+# PREP
+PRG=$( basename $0 )
+logger -t xcat -p local4.info "running '$PRG' on node $NODE"
+
+
+# DEFAULT SETTINGS
+declare -A DEFAULTS
+DEFAULTS[ENABLE]=true
```

Comment:
These `DEFAULTS` are not actually used and should probably be removed to limit confusion.